api.swagger.yaml

```diff
@@ -6369,6 +6446,183 @@ components:
+              format: uuid
+              example: 01234567-89ab-cdef-1234-56789abcdef0
+            image:
+              type: object
+              description: Avatar image of the user.
+              properties:
+                url:
+                  type: string
+                  format: uri
+                  example: >-
+                    https://dev.invotra.com/system/files/user-picture/large_avatar/1f0c6cb5-a3c3-4197-bebc-b3e44462dc24/profile.png
+            username:
+              description: The username of the user.
+              type: string
+              example: Dorothy.Whitehead@akagera.com
+            display_name:
+              description: The display name of the user.
+              type: string
+              example: Dorothy Whitehead
+            teams:
+              description: The user's teams. An array of team names and UUIDs.
+              allOf:
+                - $ref: '#/components/schemas/title-uuid-list-common-schema'
+            locations:
+              description: The user's locations. An array of location names and UUIDs.
+              allOf:
+                - $ref: '#/components/schemas/title-uuid-list-common-schema'
+            job_roles:
+              description: The user's job roles. An array of job role names and UUIDs.
+              allOf:
+                - $ref: '#/components/schemas/title-uuid-list-common-schema'
+        entity:
+          type: object
+          description: The entity that the activity is about.
+          required:
+            - uuid
+            - title
+            - body
+            - type
+          properties:
+            uuid:
+              description: The UUID of the entity.
+              type: string
+              format: uuid
+              example: 01234567-89ab-cdef-1234-56789abcdef0
+            title:
+              description: The title of the entity.
+              type: string
+              example: A post about annual leave
+            body:
+              description: The main body text of the entity.
+              type: string
+              example: I'm taking annual leave next Thursday. Cover to be announced shortly
+            type:
+              description: The entity type.
+              type: string
+              enum:
+                - node
+                - comment
+                - user
+            image:
+              description: The featured image for this entity.
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/file-schema'
+            link:
+              description: A link to the entity.
+              type: string
+              format: uri
+              example: >-
+                https://dev.invotra.com/uuid/node/1f0c6cb5-a3c3-4197-bebc-b3e44462dc26
+            parent:
+              allOf:
+                - description: The parent of the post, if the entity is a post reply (comment)
+                - $ref: '#/components/schemas/post-schema'
```

Comment:
"category": "Comment" exists within this schema.

It would be good to make this consistent with the category object but that's obviously a breaking change. It does mean we'll still be making additional calls for the parent category colour for these types of activities.

Thoughts?